selection.go

```diff
@@ -115,3 +123,58 @@ func (s *Selection) MouseToElement() error {
 
 	return nil
 }
+
+// Screenshot takes a screenshot of exactly one element
+// and saves it to the provided filename.
+// The provided filename may be an absolute or relative path.
+func (s *Selection) Screenshot(filename string) error {
+	selectedElement, err := s.elements.GetExactlyOne()
+	if err != nil {
+		return fmt.Errorf("failed to select element from %s: %s", s, err)
+	}
+
+	absFilePath, err := filepath.Abs(filename)
+	if err != nil {
+		return fmt.Errorf("failed to find absolute path for filename: %s", err)
+	}
+
+	screenshot, err := selectedElement.GetScreenshot()
+	if err != nil {
```

Comment:
Rather than adding a fallback, it could be more explicit if we have two functions, one directly use the screenshot, the other use the fallback. And after detecting the browser version, we directly calls one function or the other.